iohub/mm_fov.py

```diff
@@ -151,15 +151,20 @@ def hcs_position_labels(self):
                     row_col_fov.append(
                         ("0", match.group(4), match.group(5) + match.group(6))
                     )
-                elif match.group(7):  # "1-Pos000_000" case
+                elif match.group(7):  # "1-Pos000_000" or "1-Pos000" case
+                    optional_last_part = match.group(9) or "000"
```

Comment:
Should we always append '000'? I don't have a preference here and I'm happy with this behavior as long as it's documented.

Reply:
I went back and forth on this:

- Pro: standardizing to six digits means downstream processing can use these digits directly...at one point Ivan was writing scripts that depended on 6 digits
- Con: potential crashes e.g. `1-Pos000_000` and `1-Pos000` both map to `('0', '1', '000000'), though this will be rare because micromanager autoincrements the first number. 

I updated the documentation to make the behavior clearer. 

FWIW Jennifer and I have tested this conversion and downstream stitching on grids, lines, and a single MDA that contains boths a grid and a line.